src/test/feature/run-feature-test.sh

```diff
@@ -0,0 +1,47 @@
+#! /bin/bash
+
+if [ x$GPHOME == 'x' ]; then
+  echo "Please export GPHOME variable."
+  exit 0
+fi
```

Comment:
In the logic, user do not need to know anything about source in running feature tests. In running this script, the only thing users need to do is set GPHOME. So I think the info is enough. Thoughts?